source/_integrations/caldav.markdown

```diff
@@ -67,9 +66,9 @@ calendar:
     url: https://caldav.icloud.com
 ```
 
-This example will generate default binary sensors for each calendar you have in your account. Those calendars will be `on` when there is an ongoing event and `off` if not. Events that last a whole day are ignored in those calendars. You have to setup custom calendars in order to take them into account or for advanced event filtering.
+This example will generate default binary sensors for each calendar you have in your account. Those calendars will be `on` when there is an ongoing event and `off` if not. Events that last a whole day are ignored in those calendars.
```

Comment:
Talking about binary sensors is confusing when it's calendar entities that are meant.

Reply:
I tried to remove as much unnecessary/incorrect text as possible.